blocks/library/gallery/index.js

```diff
@@ -61,16 +70,11 @@ registerBlockType( 'core/gallery', {
 			query( 'div.blocks-gallery figure.blocks-gallery-image img', {
 				url: attr( 'src' ),
 				alt: attr( 'alt' ),
-			} ),
+			} ) || [],
```

Comment:
@aduth wouldn't we need ES6 to make everything work anyway? ES5 would bring other complications, like class variables.

@youknowriad I totally agree that better documentation and naming will go a long way. Speaking of that, looking at the current blocks, there are few two patterns that could be a bit cleaner.

- helper functions need to be both outside of the object – we can’t reference other “methods“ of the block object from `edit` or `save`, because they’re run in a totally different context. And they need to often receive `attributes` and/or `setAttributes`.
- having full-blown class components for `edit` and `save`.

If we don't go the `class` route, here’s another suggestion: switch the docs (and our current blocks) to have only variable references or short literals in the block object itself and define everything at the top:

```javascript
registerBlockType( 'core/baba', {
  title: __( 'Title' ),
  attributes,
  edit,
  save: BabaComplicatedSaveComponent, 
}
```